OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -59,7 +59,7 @@ public interface IBlocksProjectilesInfo : ITraitInfoInterface { }
 	[RequireExplicitImplementation]
 	public interface INotifyOrderIssued
 	{
-		bool OrderIssued(World world, Target target);
+		bool OrderIssued(World world, Target target, string orderString);
```

Comment:
IMO it would make sense for the orderString argument to be between world and target.